### PR TITLE
Wrote function "Nicklas"

### LINKED ISSUE #19: Write a function that adds two numbers.
In the R-Skript, write a function that takes two numbers as input and adds them.

### PR DESCRIPTION
**Pull request by burbliesj**
Closes #19

### CODE REVIEW

File: Testskript.R [hunk 1]
```diff
@@ -8,4 +8,13 @@
...
+# I want a dinosaur please! Preferably a tyrannosaurus
+
+Jule <- c(1, 2, 3, 4)
+Anne <- c(5, 6, 7, 8)
+
+Nicklas <- function(Jule, Anne) {
+  Nicklas <- (Jule + Anne)
+  return(Nicklas)
+}
+Nicklas(Jule, Anne)
```
Comment (ruckeran) [inline Testskript.R]: Wow, that looks very professional! I'm impressed!
Please change the number "8" to "9" in line 14. Thank you in advance!

File: Testskript.R [hunk 2]
```diff
@@ -8,4 +8,13 @@
...
 # :P
 
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+# I want a dinosaur please! Preferably a tyrannosaurus
+
+Jule <- c(1, 2, 3, 4)
+Anne <- c(5, 6, 7, 8)
+
+Nicklas <- function(Jule, Anne) {
```
Comment (nickhaf) [inline Testskript.R]: please give the function a more intuitive name, something like `add_numbers`. Function names should always be verbs.

**Review by ruckeran — CHANGES_REQUESTED** [review]
nice function! change request is stored in the comment

File: Testskript.R [hunk 3]
```diff
@@ -8,4 +8,13 @@
 # Hallo Jule & Nicklas, das ist unser Testskript :-) Fuer mehr Smileys hier einmal ein: "JA"
 # :P
 
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+# I want a dinosaur please! Preferably a tyrannosaurus
+
+Jule <- c(1, 2, 3, 4)
+Anne <- c(5, 6, 7, 8)
```
Comment (nickhaf) [inline Testskript.R]: These are example code lines, please put them below the function.

File: Testskript.R [hunk 4]
```diff
@@ -8,4 +8,13 @@
 # Hallo Jule & Nicklas, das ist unser Testskript :-) Fuer mehr Smileys hier einmal ein: "JA"
```
Comment (nickhaf) [inline Testskript.R]: Please remove these comments.

File: Testskript.R [hunk 5]
```diff
@@ -8,4 +8,13 @@
...
 # :P
 
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+# I want a dinosaur please! Preferably a tyrannosaurus
```
Comment (nickhaf) [inline Testskript.R:2]: This one as well.

**Review by nickhaf — CHANGES_REQUESTED** [review]
Works great, thanks! 
Some minor naming issues and clean up ideas are commented in the script.

File: Testskript.R (hunk 1 above)
Comment (nickhaf) [reply Testskript.R]: You can also put the comment directly below line 14 by clicking on the blue plus at line 14 :)

Comment (ruckeran) [reply Testskript.R]: ah nice, good to know!

File: Testskript.R (hunk 5 above)
Comment (burbliesj) [reply Testskript.R:2]: Thanks for the suggestion, but I would like to keep some fun in this script!

File: Testskript.R (hunk 4 above)
Comment (burbliesj) [reply Testskript.R]: Will do

File: Testskript.R (hunk 3 above)
Comment (burbliesj) [reply Testskript.R]: Thanks for your feedback!

File: Testskript.R (hunk 1 above)
Comment (burbliesj) [reply Testskript.R]: Thanks for the feedback, already adjusted the script!

File: Testskript.R (hunk 5 above)
Comment (nickhaf) [reply Testskript.R:2]: okay thats fine :D

Comment (ruckeran) [reply Testskript.R:2]: i think that's a great addition to the script, too

File: Testskript.R [hunk 6]
```diff
@@ -1,11 +1,12 @@
...
 
+Jule <- c(1, 2, 3, 4)
+Anne <- c(5, 6, 7, 9)
 
-# Hallo Jule & Nicklas, das ist unser Testskript :-) Fuer mehr Smileys hier einmal ein: "JA"
-# :P
-
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+Nicklas(Jule, Anne)
```
Comment (nickhaf) [inline Testskript.R]: Great work! Please use the new function name instead of `Nicklas`. The example won't work otherwise.

**Review by nickhaf — CHANGES_REQUESTED** [review]
Great work, thanks! I requested one minor addition.

**Review by ruckeran — APPROVED** [review]
Awesome work! Was fun working with you!

File: Testskript.R (hunk 6 above)
Comment (burbliesj) [reply Testskript.R]: Thanks for the reminder, I blame that mistake on being in a hurry at the end :)

**Review by nickhaf — APPROVED** [review]
Fantastic work. Please merge.

**Pull request merged**